shared/attributes.asciidoc

```diff
@@ -87,6 +87,7 @@ endif::[]
 :endpoint-guide:       https://www.elastic.co/guide/en/endpoint/{branch}
 :sql-odbc:             https://www.elastic.co/guide/en/elasticsearch/sql-odbc/{branch}
 :ecs-ref:              https://www.elastic.co/guide/en/ecs/{ecs_version}
+:ecs-logging-ref:          https://www.elastic.co/guide/en/ecs-logging/{ecs-logging}
```

Comment:
Not sure if you meant for this to be indented more than the others.

Reply:
🤦 Now I get to wait another 20 mins for the build to pass again

Reply:
😂